cli/types/types.go

```diff
@@ -54,8 +54,8 @@ type ZarfComponent struct {
 	// Scripts are custom commands that run before or after package deployment
 	Scripts ZarfComponentScripts `yaml:"scripts,omitempty"`
 
-	// Import refers to another zarf.yml package.
-	Import string `yaml:"import,omitempty"`
+	// Import refers to another zarf.yaml package.
+	Import ZarfImport `yaml:"import,omitempty"`
```

Comment:
Curious where you think this is going to need a separate struct.  Total fine if it's needed, just like to understand if you see there being more than just a Path element in the future.

Reply:
I was thinking so we don’t have to change everything later for pulling from external, but I really wanted validation on that one cause I wasn’t sure if it would be needed. I figured it would leave room for a sha with out introducing a breaking change later. Could just be getting ahead though and can revert.

Reply:
Also if we want to make this extensible and map overrides it will be pretty simple to add something like import.Components to override the imported packages component fields where there are updates.